budgetApp1/budgetApp1/LoginViewController.swift

```diff
@@ -2,21 +2,21 @@ import UIKit
 
 class LoginViewController: UIViewController {
     
-    // MARK: - IBOutlets
+    // MARK: - variables
     @IBOutlet private weak var username: UITextField!
     @IBOutlet private weak var password: UITextField!
     
-    let userViewModel = UserViewModel.shared // Use shared instance
-    private var loggedInUser: User?
+    private lazy var userViewModel = UserViewModel(authenticationRepository: AuthenticationRepository())
+    
+    private var loggedInStatus = false
     
```

Comment:
hey bro, just double check that  your // MARKs, they're a bit disordered here, also check unnecessary spacing 💪

Reply:
will do